packages/esm-patient-forms-app/src/index.ts

```diff
@@ -39,6 +39,10 @@ export const patientHtmlFormEntryWorkspace = getAsyncLifecycle(
 
 // t('clinicalForms', 'Clinical forms')
 export const clinicalFormsWorkspace = getAsyncLifecycle(() => import('./forms/forms-dashboard.workspace'), options);
+export const clinicalFormsWorkspaceExtension = getAsyncLifecycle(
+  () => import('./forms/forms-dashboard.workspace'),
+  options,
+);
```

Comment:
This is a duplicate declaration; it doesn't add anything. The code would be easier to understand if the extension registration simply referred to the `clincialFormsWorkspace` object. Then it would be immediately obvious we are doing something weird, rather than taking time to figure out we are doing something weird :)